collector/src/main/java/io/prometheus/jmx/JmxScraper.java

```diff
@@ -116,7 +116,9 @@ public void doScrape() throws Exception {
                 mBeanNames.removeAll(beanConn.queryMBeans(name, null));
             }
             for (ObjectInstance name : mBeanNames) {
+                long start = System.nanoTime();
                 scrapeBean(beanConn, name.getObjectName());
+                logger.fine("TIME " + (System.nanoTime() - start) + " : " + name.getObjectName().toString());
```

Comment:
Can you put the unit in here? You shouldn't have to read the code to see it's ns